src/AutoNumericOptions.js

```diff
@@ -282,6 +282,8 @@ Object.defineProperty(AutoNumeric, 'options', {
                 focus : 'focus',
```

Comment:
Please add a reference to the new options `min` and `max` in the JSDoc of this `emptyInputBehavior` option